<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>CPU</string>
       </attribute>
       <widget class="QWidget" name="cpuwidget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>821</width>
          <height>491</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>510</y>
          <width>821</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>cache size:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>frequency:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>cpu-utilization</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="cachesizelabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="freqlabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cpulabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>memory</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>801</width>
          <height>23</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>memory total</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="memlabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>801</width>
          <height>511</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>memory free</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="MemFreebar">
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>memory available</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="MemAvailblebar">
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Buffers</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="bufbar">
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>cached</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="cachedbar">
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>network</string>
       </attribute>
       <widget class="QWidget" name="netwidget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>811</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>590</y>
          <width>791</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>recv pkg</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="rcvlabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>trans pkg</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="tranlabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>recv Byte</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="rcvlabel_2">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>trans Byte</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="tranlabel_2">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>544</y>
          <width>791</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#204a87;&quot;&gt;up speed:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ulabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#a40000;&quot;&gt;downspeed:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="dlabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>process</string>
       </attribute>
       <widget class="QTextBrowser" name="procsshower">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>36</y>
          <width>671</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>656</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>USER  PID  CPU  MEM  VSZ  RSS   TTY   STAT    START   TIME           COMMAND</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>250</y>
          <width>106</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>MEM</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>300</y>
          <width>106</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>200</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Sort by</string>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>791</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>530</y>
          <width>791</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>550</y>
          <width>801</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>process total</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pronumlabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>process running</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="prorunlabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>process blocked</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="problklabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>380</y>
          <width>801</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>input pid:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>kill</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>440</y>
          <width>621</width>
          <height>99</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>PID   TTY      STAT             TIME                              COMMAND</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="procsshower_2"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>about</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>821</width>
          <height>621</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>os version</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="oslabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>kernel version</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="kernellabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>model name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="modelnamelabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>system time:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="modelnamelabel_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_23">
           <item>
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>run time:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="modelnamelabel_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>650</y>
          <width>106</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>shutdown</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
   <widget class="QLabel" name="rlabel3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rlabel2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>460</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rlabel1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>420</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
